Dockerfile

```diff
@@ -0,0 +1,20 @@
+ARG PYTHON_VERSION=3.8
+
+FROM python:${PYTHON_VERSION}
+
+ENV ENV="Local"
+ENV PYTHONUNBUFFERED=1
+
+WORKDIR /app
+
+COPY requirements.txt .
+
+RUN pip install -r requirements.txt
+
+COPY . .
```

Comment:
Copying the entire context with `COPY . .` can lead to larger image sizes if there are unnecessary files in the build context. Ensure your `.dockerignore` file is correctly set up to exclude unnecessary files.

Reply:
I do exactly like my mentor show me. It is work fine